lambdatimes/src/components/Content/Content.js

```diff
@@ -18,13 +18,22 @@ export default class Content extends Component {
 
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
+    this.setState ({
+      tabs: tabData,
+      cards: cardData
+    })
   }
 
   changeSelected = tab => {
     // this function should take in the tab and update the state with the new tab.
+    this.setState ({
+      selected: tab
+    })
   };
 
   filterCards = () => {
+    if (this.state.selected === 'all'){return this.state.cards;}
+    else{ return this.state.cards.filter(card => card.tab === this.state.selected)}
```

Comment:
break these up over more lines, a curly space block should indicate a line break